apps/website/src/pages/docs/datepicker.mdx

```diff
@@ -14,44 +14,74 @@ import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-datepicker' />
-
 <LiveExample src='DatePicker.main.tsx' truncate={false}>
   <AllExamples.DatePickerMainExample client:load />
 </LiveExample>
 
 When a user needs to schedule deadlines, timeframes, or for age verification purposes, we provide them with a date picker. It behaves much like a select menu. We currently have three different types of date pickers.
 
-## Appearance
+## Variants
 
-### Default
+### Basic
 
-The default date picker is a default menu button from which an interactive calendar expands on click. This allows the user to visually select the date rather than entering the date with a keyboard. This has the advantage of also showing weekdays, which may be important in a given context. The date picker is commonly used within column filters.
+The basic date picker allows the user to pick a date from the calendar and skip a month at a time.
 
-### Text field
+<LiveExample src='DatePicker.basic.tsx' truncate={true}>
+  <AllExamples.DatePickerBasicExample client:load />
+</LiveExample>
 
-There are two ways to select a date with the text field date picker; clicking on the button and selecting a date, or typing it out using the format shown in the help message underneath the field (DD-MM-YYYY). The date automatically fills the text field when directly selected in the calendar and is editable. It’s a more convenient way to offer editing in a context where the user might need to change the date often, or enter different dates for several items in a workflow.
+This allows the user to visually select the date rather than entering the date with a keyboard. This has the advantage of also showing weekdays, which may be important in a given context. The date picker is commonly used within column filters.
```

Comment:
```suggestion
This allows the user to visually select the date rather than entering the date with a keyboard. This has the advantage of also showing weekdays, which may be important in a given context. The date picker is commonly used within column filters in Table.
```
Also make sure to link `Table` to the Table documentation. There's an example of how to do this in the Expandable Block documentation page

Reply:
done

Reply:
Make sure it links to the Table documentation like so

```mdx
[Table](table)
```

Reply:
Fixed!